charts/backbeat/templates/replication/status_deployment.yaml

```diff
@@ -44,6 +44,10 @@ spec:
               value: "{{- printf "%s-%s" .Release.Name "redis" | trunc 63 | trimSuffix "-" -}}"
```

Comment:
Both REDIS and REDIS_LOCALCACHE are pointing to the same Redis Service. Ideally the REDIS one should be pointing to Redis-HA service (if it's even needed). This was a misconfiguration from before and should be fixed.